packages/react-wildcat-handoff/src/utils/serverContext.js

```diff
@@ -5,21 +5,29 @@ const createReactClass = require("create-react-class");
 
 const RouterContext = Router.RouterContext;
 
-module.exports = function serverContext(request, headers, renderProps) {
+module.exports = function serverContext(
+    {request, response}, // eslint-disable-line react/prop-types
```

Comment:
This looks like a change in argument signature. Do you also need to change `simpleRender.js` to include the `response`?